package.json

```diff
@@ -34,7 +34,7 @@
     "@actions/core": "^1.10.0",
     "@actions/github": "^4.0.0",
     "actions-secret-parser": "^1.0.4",
-    "azure-actions-appservice-rest": "^1.3.3",
+    "azure-actions-appservice-rest": "^1.3.6",
```

Comment:
Is this upgrade intentional?

Reply:
yes, we published the new version on our dependent repo.